filebeat/modules.d/logstash.yml.disabled

```diff
@@ -0,0 +1,16 @@
+- module: logstash
+  # logs
+  logs:
+    enabled: true
+
+    # Set custom paths for the log files. If left empty,
+    # Filebeat will choose the paths depending on your OS.
+    #var.paths:
+
+  # Slow logs
```

Comment:
It's a bit strange that we call it slow logs in the comments but below it's error?

Reply:
oops, typo good catch :)

Reply:
I will do another complete run, are you volunteering to test it? 💃

Reply:
can do, but probably only get to it tomorrow.